.github/workflows/CypressImage.yaml

```diff
@@ -37,14 +37,14 @@ jobs:
           cp -r /home/runner/.gradle ./gradlecache
           cp -r /home/runner/.cache/Cypress ./cypresscache
       - name: Login to Github Packages
-        uses: docker/login-action@v2
+        uses: docker/login-action@v3
```

Comment:
Soweit ich das sehen konnte gibt es login-action nur in v2?!
Aber im Code gibt es noch eine Stelle wo v1 benutzt wird.

Reply:
beides angepasst